opencontainers/struct.py

```diff
@@ -293,15 +303,40 @@ def add(self, name, value):
         """
         Add a value to an existing attribute, normally when used by a client
         """
+        if value is None:
+            return
         if name not in self.attrs:
             bot.exit("%s is not a valid attribute." % name)
-        attr = self.attrs[name]
-
-        # If we already have a value, update it
-        if attr.value and isinstance(attr.value, dict):
-            value.update(attr.value)
-        elif attr.value and isinstance(attr.value, list):
-            value = list(set(attr.value + [value]))
+        attr: StructAttr = self.attrs[name]
```

Comment:
What versions of Python will this syntax not be supported by?

Reply:
Typing hinting started in python 3.5.
I've just removed the type hint.
My IDE was not showing hints for `attr`, so it was accidentally commited..